hw03_Nyabutoa.py

```diff
@@ -0,0 +1,67 @@
+
+#################################################################################
+# Author: Alpha Nyabutop
+# Username: Nyabutoa
+#
+# Assignment:HW03: Fully Functional Gitty Psychedelic Robotic Turtles
+# Purpose:
+# Google Doc Link:https://docs.google.com/document/d/1A_q6VWbkQ8u_noD3prdv25tmySfbxk-JVBYkq4vUlPI/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+#
+#
+#################################################################################
+
+
+import turtle
+
+
+def circlec(nyabutoa):
+    nyabutoa.goto(0,0)
+    r=180
+    nyabutoa.pensize(10)
+    nyabutoa.circle(r)
+    nyabutoa.left(90)
+    nyabutoa.penup()
+    nyabutoa.forward(r+50)
+    nyabutoa.left(90)
+    nyabutoa.forward(100)
+    nyabutoa.pendown()
+    eyez(nyabutoa)
+    nyabutoa.right(90)
+    nyabutoa.penup()
+    nyabutoa.forward(20)
+    nyabutoa.right(90)
+    nyabutoa.forward(200)
+    nyabutoa.left(180)
+    nyabutoa.pendown()
+    eyez(nyabutoa)
```

Comment:
Why did you call eyez from inside this function, instead of from main where it's commented out?

A function should do a single job (e.g., `draw_eye` should draw eyes, and nothing else). So if your intent is that the first function is `draw_head` and a head includes drawing two eyes, then this structure could make sense. However, if `eyez` draws something that is not related to the `circlec` function, then this structure is not correct. Bad variable and function names and no docstrings makes it unclear which case is true.

Reply:
I called the 'eyez' function from inside the circle function because I needed the eyes to be drawn inside the head.